pgbouncer/datadog_checks/pgbouncer/pgbouncer.py

```diff
@@ -73,47 +74,46 @@ def _collect_stats(self, db):
             metric_scope.append(SERVERS_METRICS)
 
         try:
-            with db.cursor(cursor_factory=pgextras.DictCursor) as cursor:
```

Comment:
PG3 `with` automatically starts a transaction, i.e., adds a `BEGIN`, which breaks many of the queries.